openssl/bbig/big.go

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// Copyright 2022 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// This is a mirror of crypto/internal/boring/bbig/big.go.
```

Comment:
https://github.com/golang/go/blob/b2116f748a7acd32facdbbdaf40117ed40564083/src/crypto/internal/boring/bbig/big.go is just `Enc` and `Dec`, but this file has `GenerateKeyECDSA`, `GenerateKeyRSA` etc. too.

Should this file be split into `big.go` as a pure mirror, plus another file (say, `wrappers.go`) for the converting wrappers? (And would it let us split up the license header between upstream and our code?)

Reply:
I've gone a little bit further and move all the wrapper to `openssl/bbig/bridge` so it is clear what those wrappers are for. Additionally, the `encoding/asn1` import in `bbig` breaks go1.19 `TestDependencies`, so removing that will have minimizing diffs.